staNMF/staNMF.py

```diff
@@ -134,16 +133,16 @@ def load_data(self):
             try:
                 csvfile = open(self.fn, "r")
             except:
-                ("File ''" + str(self.fn) + "' not found.")
+                print("File ''" + str(self.fn) + "' not found.")
```

Comment:
consider using loggers in the future. allows people who import to customize the output